src/main/java/org/folio/consortia/repository/UserTenantRepository.java

```diff
@@ -17,8 +18,16 @@ public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UU
   @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.username= ?1 AND ut.tenant.id= ?2")
   Optional<UserTenantEntity> findByUsernameAndTenantId(String username, String tenantId);
 
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.tenant.id= ?2")
```

Comment:
is there an index for these fields already?

Reply:
Yes
![image](https://user-images.githubusercontent.com/113523904/231149418-512dd00d-1cdb-46e1-b458-fc998aa87b86.png)